zulipterminal/helper.py

```diff
@@ -529,6 +530,21 @@ def format_string(names: List[str], wrapping_text: str) -> List[str]:
     return [wrapping_text.format(name) for name in names]
 
 
+def streams_view_divider() -> Any:
```

Comment:
This is now basically a factory function which initializes a divider in another way - so maybe let's just make this a simple subclass?